tools/release/common.sh

```diff
@@ -38,5 +38,10 @@ find_prev_release() {
     exit 1
   fi
 
+  # Find the prev release version. Since all tags start with "mimir-"
+  # we can simply find the version:
+  PREV_RELEASE_VERSION=$(echo "${PREV_RELEASE_TAG}" | cut -c 7-)
```

Comment:
I know this is after the merge but I managed to not post the comment that was part of my review.

Another way we can achieve the same thing is with: `PREV_RELEASE_VERSION="${PREV_RELEASE_TAG#mimir-}"`

```console
printf "    TAG: %s\nVERSION: %s\n" "$PREV_RELEASE_TAG" "${PREV_RELEASE_TAG#mimir-}"

    TAG: mimir-2.4.0-rc.0
VERSION: 2.4.0-rc.0
```